testing/benchmark/pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: flutter_engine_benchmark
+
+dependencies:
+  git: any
+  metrics_center:
+    # TODO(liyuqian): once metrics_center is properly reviewed, add it to
+    # flutter/packages, publish on pub.dev, and use the published package here.
+    git: https://github.com/liyuqian/metrics_center.git
```

Comment:
What does metrics center do and why not integrating parse_and_send.dart into metrics center?